src/redux/Graphql/saga.js

```diff
@@ -131,6 +280,10 @@ export function* getOraclesHandler() {
         value: [],
         limit: action.limit,
         skip: action.skip,
+        error: {
```

Comment:
these background graphql calls dont need error objects since we fail silently, remember?